### PR TITLE
[Kitsu] Add ruleset

### PR DESCRIPTION
**Pull request by synthtech**

### CODE REVIEW

File: src/chrome/content/rules/Kitsu.io.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
+	Not resolved:
+		- www.kitsu.io
+
+-->
+<ruleset name="Kitsu (partial)">
```
Comment (J0WI) [inline src/chrome/content/rules/Kitsu.io.xml]: Why is this rule partial?

Comment (synthtech) [inline src/chrome/content/rules/Kitsu.io.xml]: Since `www` wasn't working I thought I should label it as partial. I can change it if it's not needed.

Comment (J0WI) [inline src/chrome/content/rules/Kitsu.io.xml]: Referig to your comment www does not exist, so the host is fully covered with those rules.

Comment (synthtech) [inline src/chrome/content/rules/Kitsu.io.xml]: Ok, I changed it then

**Pull request merged**